src/Panel/JobPanel.php

```diff
@@ -19,7 +19,7 @@ public function summary()
     }
 
     /**
-     * {@inheritDoc}
+     * {@inheritdoc}
```

Comment:
no problem, if styleci is wrong it can easily be changed afterwards. Let's first fix all this, correcting will be easy after that.